src/data/events.json

```diff
@@ -0,0 +1,21 @@
+{
+	"events": [
+		{
+			"type": "social",
+			"title": "Casual Catchup",
+			"date": "2018-08-08T10:00:00.000Z",
+			"url": "https://www.meetup.com/en-AU/Front-End-Web-Developers-Perth/events/qqvhjkyxlblb/",
+			"active": true
```

Comment:
It's possible this isn't needed and you could just add it to the first one I just wasn't sure what it was doing

Reply:
@mandymichael may be able to confirm about this